test/controllers/observations_controller/observations_controller_show_test.rb

```diff
@@ -688,7 +688,7 @@ def test_prev_and_next_observation_with_fancy_query
       :Observation, names: { lookup: n2.id, include_synonyms: false },
                     by: :name
     )
-    assert_equal(2, query.num_results)
+    assert_equal(1, query.num_results)
```

Comment:
The comment above this line should be maybe altered. I believe it's indicating that a misspelling should get you the correct spelling and the misspelling.

Reply:
Done.